### ISSUE TITLE
Facebook Poxel

### ISSUE THREAD

**Issue by masoudilyas** [facebookincubator/facebook-for-magento#69]
Hi, this is a great extension. I read in your FAQs that if someone has already installed the Facebook pixel, then they must remove that pixel, and use the extension to install the pixel. I have the pixel installed through Google Tag Manager, and all of the conversion events. My question is that is there any functionality in the extension to skip the Installation of the pixel using the extension, and only use the rest of the functionality in the app, for example the Facebook shop creation and most importantly the Facebook Catalog functionality. Sorry in advance if this is the wrong place to ask this question, i couldn't find any support page. Thank you.
Kind Regards
Masoud

**Comment by dmitridr** [comment]
Hi @masoudilyas 

Thanks for your interest in our plugin! 
Regarding your question, unfortunately there is no easy way to do what you ask, but there is a way.

Check out Issue #40 , you can use the DB editing procedure outlined in that issue to disable the pixel (by deleting the entry mentioned in that issue). Though your site may have JavaScript console warnings as a result. If those bother you, you may have to dig into the code of the extension on your server to disable those manually by removing [the following lines](https://github.com/facebookincubator/facebook-for-magento/blob/v2.5.0/app/design/frontend/base/default/template/facebook_ads_extension/head.phtml#L24). 

I would also make sure the pixel installed with Google Tag Manager is correctly attached to the catalog created by the extension, if you plan to run dynamic ads! Hope that helps.

**Comment by masoudilyas** [comment]
Thanks for your reply. I will try that. But i suggest that you can make that a functionality in the extension, in future releases, because there are a lot of people who already have spent a lot of time and effort installing the conversion tags using Google tag manager, because the level of customisation is higher doing it with GTM. 
Kind Regards
Masoud

**Comment by dmitridr** [comment]
Good idea, I will mark this as a feature request, we can put a checkbox under advanced options.

**Comment by masoudilyas** [comment]
Hi dmitridr, I am glad that you have marked this as a feature request. Do you have any time frame, as to when this feature will be implemented, as i am not comfortable deleting database files, i am a novice when it comes to development.
Kind Regards
Masoud

**Comment by dmitridr** [comment]
Not very soon, I am the only developer on this plugin right now, and my time is split between other work. If you or any others reading this want to jump in and build this feature, we will welcome your pull request!

I would say if there is no pull request, there's a good chance we can have this before the end of the year.

If you are not comfortable editing database files, I recommend editing the extension code itself, which should be less risky. If you can find [this line of code](https://github.com/facebookincubator/facebook-for-magento/blob/master/app/code/community/Facebook/AdsExtension/Block/common.php#L48) in the extension files on your server, simply change it to `return 0;` it should disable the pixel.

**Comment by MartinKonVis** [comment]
hello @dmitridr Thanks for the infos. Unfortunatly the link to the code is not longer working. Could you please provide the path or filename?
